.travis.yml

```diff
@@ -27,10 +27,10 @@ jobs:
       after_success:
     - stage: test
       env:
-        - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1"
+        - TESTS="1" ODOO_REPO="odoo/odoo" MAKEPOT="1" EXCLUDE="sale_warehouse_calendar"
     - stage: test
       env:
-        - TESTS="1" ODOO_REPO="OCA/OCB"
+        - TESTS="1" ODOO_REPO="OCA/OCB" EXCLUDE="sale_warehouse_calendar"
```

Comment:
Can you add a comment about the conflicting module?

Reply:
done in the doc.

Reply:
are you sure you are running your tests w/ this settings?